python-requre.spec

```diff
@@ -0,0 +1,81 @@
+# Created by pyp2rpm-3.3.3
+%global pypi_name requre
+
+Name:           python-%{pypi_name}
+Version:        0.2.1.dev1+gf480bd7.d20200120
+Release:        1%{?dist}
+Summary:        Library for testing python code what allows store output of various objects and use stored data for testing
+
+License:        MIT
+URL:            https://github.com/packit-service/requre
+Source0:        requre-0.2.1.dev1+gf480bd7.d20200120.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3dist(click)
+BuildRequires:  python3dist(pytest)
+BuildRequires:  python3dist(pyyaml)
+BuildRequires:  python3dist(requests)
+BuildRequires:  python3dist(setuptools)
+BuildRequires:  python3dist(setuptools-scm)
+BuildRequires:  python3dist(setuptools-scm-git-archive)
+BuildRequires:  python3dist(sphinx)
+
+%description
+ REQUest REcordingRequre \[rekure\]Is Library for storing output of various
+function and methods to persistent storage and be able to replay the stored
+output to functions [Documentation]( Plan and current status - Used for testing
+[packit-service]( organization projects - ogr - packit
+
+%package -n     python3-%{pypi_name}
+Summary:        %{summary}
+%{?python_provide:%python_provide python3-%{pypi_name}}
+
+Requires:       python3dist(click)
+Requires:       python3dist(pytest)
+Requires:       python3dist(pyyaml)
+Requires:       python3dist(requests)
+Requires:       python3dist(setuptools)
```

Comment:
All the `Requires:` should not be needed since Fedora now (F30 and newer) [automatically generates dependencies](https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_automatically_generated_dependencies)